src/completions/Excmd.ts

```diff
@@ -90,6 +90,15 @@ export class ExcmdCompletionSource extends Completions.CompletionSourceFuse {
             }
         }
 
+        // Add partial matched funcs like: 'conf' ~= 'viewconfig'
```

Comment:
I've asked for some comments from users on our chatroom before I merge this change as I think it might be a little controversial :)

Reply:
It's not like it will auto complete for you. User still need to tab to complete. 

It's just a helper to remind some cmds which don't have straight forward prefix.

Reply:
I hadn't realised that they would appear at the bottom - so I'm fine with this and 100% of users on Riot agreed (N=1).